src/EditorFeatures/Core/IntelliSense/AsyncCompletion/ItemManager.cs

```diff
@@ -120,13 +135,28 @@ private static SegmentedList<VSCompletionItem> SortCompletionItems(AsyncCompleti
                     }
                 }
 
-                var updater = new CompletionListUpdater(session.ApplicableToSpan, sessionData, data, _recentItemsManager, _globalOptions);
+                var updater = new CompletionListUpdater(session.ApplicableToSpan, sessionData, data, _recentItemsManager, _editorOptionsService.GlobalOptions);
                 return await updater.UpdateCompletionListAsync(session, cancellationToken).ConfigureAwait(false);
             }
             finally
             {
                 AsyncCompletionLogger.LogItemManagerUpdateDataPoint(stopwatch.Elapsed, isCanceled: cancellationToken.IsCancellationRequested);
             }
+
+            // Don't hide expanded items if all these conditions are met:
+            // 1. filter text length >= 2 (it's common to use filter of length 2 for camel case match, e.g. `AB` for `ArrayBuiler`)
+            // 2. the completion is triggered in the context of listing members (it usually has much fewer items and more often used for browsing purpose)
+            // 3. defaults is not empty (it might suggests an expanded item)
```

Comment:
i don't understand #3.  can you expand on that more.  Primarily, i don't know what `defaults` actually means in the context of completion.